src/components/application_manager/include/application_manager/application_manager_impl.h

```diff
@@ -1368,6 +1380,17 @@ class ApplicationManagerImpl
   sync_primitives::Lock audio_pass_thru_lock_;
   sync_primitives::Lock tts_global_properties_app_list_lock_;
   hmi_apis::Common_DriverDistractionState::eType driver_distraction_state_;
+
+  /**
```

Comment:
@IGapchuk wrong indents

Reply:
@AByzhynar Fixed in 0bb0e987107bdb5669927ed6977a81157f0fbad8.